kokoro-linux.sh

```diff
@@ -0,0 +1,20 @@
+# Install NVM
```

Comment:
We'll need the copyright blurb (that's bureaucracy), and #!/bin/sh (that's more about making this work properly).